### ISSUE TITLE
Support status bar dynamic theming for the iOS PWA

### ISSUE THREAD

**Issue by agneevX** [6c65726f79/Transmissionic#1157]
Currently, A2HS on iOS has a black status bar. The site header is also not `#000000`, so it looks a little out of place.

From Mozilla Web Docs (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/meta/name/theme-color), the `theme-color` tag can be added...

```html
<meta
  name="theme-color"
  media="(prefers-color-scheme: light)"
  content="white" />
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="black" />
```

Per this SO answer, it should be possible to achieve this by removing the `apple-mobile-web-app-status-bar-style`: https://stackoverflow.com/a/69699288/12144618 as it seems to supersede the `theme-color` tag.

**Comment by agneevX** [comment]
With the following, I was able to get the default scheme to display properly:

```html
<meta name="theme-color" media="(prefers-color-scheme: light)" content="#f6f7f9" />
<meta name="theme-color" media="(prefers-color-scheme: dark)" content="#0a0a0a" />

<meta name="apple-mobile-web-app-status-bar-style" content="default"/>
```

<img src="https://user-images.githubusercontent.com/19761269/194841865-9c69896d-d8c5-422d-a7b7-8a9d0af62d35.jpg" width="400">

<img src="https://user-images.githubusercontent.com/19761269/194841802-b0cc8e33-f14b-4e5b-b2dc-86d166966f7e.png" width="400">

**Comment by 6c65726f79** [comment]
Hi, I have already tried to solve this issue, but unfortunately there is no way to dynamically update the color of the status bar according to the user's preferences.

I'll add the code of your second message anyway, it's better than nothing. Thank you for sharing it!

**Comment by 6c65726f79** [comment]
Available in [v1.7.1](https://github.com/6c65726f79/Transmissionic/releases/tag/v1.7.1).
